src/pages/data-management/index.jsx

```diff
@@ -31,23 +34,25 @@ const DataManagementPage = () => {
   const [newProjectModalVisible, setNewProjectModalVisible] = useState(false);
 
   useEffect(() => {
+    if (privacyPolicyIsNotAccepted(user, domainName)) return;
```

Comment:
I won't be a problem because the real block is happening in the api, this is just to be nice and avoid having the user see an error message from the api instead